app/components/lg-cv-result-summary.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  classNames: ['lg-cv-result-summary'],
+  cvResult: null,
+  action: null,
```

Comment:
I'm iffy on whether `action` really needs to be specified here since it is part of the standard api for an ember component. I guess it's fine, and I like the pattern of clarifying expected inputs, which is what these `null`s are for.